crypto-volatility-index/src/cryptoVolatilityIndex.js

```diff
@@ -0,0 +1,100 @@
+const { logger } = require('@chainlink/external-adapter')
+const Big = require('big.js')
+const dataProvider = require('./derivativesDataProvider.js')
+const marketCapProvider = require('./marketCapProvider.js')
+const sigmaCalculator = require('./sigmaCalculator.js')
+const cron = require('node-cron')
+const moment = require('moment')
+const { POINTS_TO_AVG, FREQ_MINUTES } = require('./config')
+const cryptoCurrencies = ['BTC', 'ETH']
+const cvxValues = {}
+
+const calculate = async () => {
+  // Get all of the required derivatives data for the calculations, for all the relevant currencies
+  const derivativesData = await dataProvider.getDerivativesData(cryptoCurrencies)
+  // Calculate vix values for all currencies
+  const volatilityIndexData = await calculateVixValues(derivativesData)
+  // Apply averaging based on previous iterations
+  const smoothedData = await applyAveraging(volatilityIndexData)
+  // Apply weights to calculate the Crypto Vix
+  const cvx = await calculateWeighted(smoothedData)
+  logger.info(`CVX: ${cvx}`)
+  return cvx
+}
+
+let _isScheduled = false
+const scheduleCron = async () => {
+  if (_isScheduled) return
+
+  cron.schedule(`0 */${FREQ_MINUTES} * * * *`, async () => {
+    await calculate()
+  })
+}
```

Comment:
This seems unnecessary to me. On `execute()` we are calling `cryptoVolatilityIndex.calculate()` anyways. Is there a reason for running calculate() on a schedule too?

Reply:
the schedule was added in order to support averaging, after review - it isn't required and was removed